docs/DevGuide/OptionParsing.md

```diff
@@ -2,7 +2,7 @@
 Distributed under the MIT License.
 See LICENSE.txt for details.
 \endcond
-# Option Parsing {#tutorial_option_parsing}
+# Option Parsing {#option_parsing}
```

Comment:
no

Reply:
Why is this file being moved?

Reply:
Figured it's more of a tutorial for developers and not for users, so moved it to _Dev Guide_.

Reply:
Okay, is it still being pointed to from the Tutorials one? We should probably have a prefix for dev guide things too because I worry that the refs will conflict. Maybe just `dev_guide_option_parsing`?

Reply:
The link from the _User Tutorials_ page I removed.